src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: local("Avenir"),
+    url("fonts/Avenir.ttc") format("ttc")
+    url(fonts/Avenir-Book.ttf) format("ttf")
+    url(fonts/Avenir-Heavy.ttf) format("ttf");
+}
+
+.search-bar2 {
+  width: 100%;
+  margin-top: 20px;
+  line-height: 19px;
+}
+
+.search-bar {
+  width: 100%;
+  margin-top: 20px;
+  line-height: 22px;
+}
```

Comment:
Do not repeat your self
![2023-04-28_14h35_10](https://user-images.githubusercontent.com/61904995/235137280-cb88873a-e899-4774-bd93-e7ea7f990836.png)